routes/adminRoutes.js

```diff
@@ -84,6 +136,38 @@ router.delete('/org/:orgId', async (req, res) => {
   }
 });
 
+router.put('/org/:orgId', async (req, res) => {
+  try {
+    const id = req.params.orgId;
+    const { shortName, fullName } = req.body;
+    Exceptions.throwIfNull({ id, shortName, fullName });
+    if (shortName === undefined || fullName === undefined) {
```

Comment:
why are you checking this? null check will throw an exception anyways

Reply:
Oops, forgot to remove it